### PR TITLE
show which topic this question belongs to

### PR DESCRIPTION
**Pull request by vvs79**

### CODE REVIEW

File: app/controllers/admin/questions_controller.rb [hunk 1]
```diff
@@ -9,6 +9,9 @@ def index
 
   def show
     @question = Question.find(params[:id])
+    @topic = Topic.find_by_id(@question.topic_id)
+    @t_q = @topic.title
+    @t_q = 'None' if @topic.title.nil?
```
Comment (Kmalynss) [inline app/controllers/admin/questions_controller.rb]: - variable naming is bad
- topics are not allowed without title, so no need of this check
- no need of @topic and @t_q variable

File: app/views/admin/questions/show.html.slim [hunk 2]
```diff
@@ -11,5 +11,10 @@ div
     strong
       | Answer:
     = @question.answer
+div
+  p.center
+    strong
+      | Topic:
+    = @t_q
```
Comment (Kmalynss) [inline app/views/admin/questions/show.html.slim]: this must be shown via ```@question``` i.e.:
```ruby
@question.topic.title
```

File: app/controllers/admin/questions_controller.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@ def index
 
   def show
     @question = Question.find(params[:id])
+    @question.topic = Topic.find_by_id(@question.topic_id)
```
Comment (Kmalynss) [inline app/controllers/admin/questions_controller.rb]: this is not needed
please explain why

**Comment by vvs79** [comment]
because questions and topics related to each other,
in model "question.rb"  there is "belongs_to :topic"
in model "topic.rb"  there is "has_many :questions"
association one to many

**Comment by vvs79** [comment]
https://trello.com/c/UmMqsA69/32-on-a-question-page-show-which-topic-this-question-belongs-to

**Pull request merged**